src/Bookings.js

```diff
@@ -1,18 +1,47 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
+import SearchResults from "./SearchResults.js";
 // import FakeBookings from "./data/fakeBookings.json";
+import CustomerProfile from "./CustomerProfile.js";
 
 const Bookings = () => {
+  const [booking, setBooking] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(false);
+  const [customerProfile, setCustomerProfile] = useState([]);
+
+  useEffect(() => {
+    fetch(`https://cyf-react.glitch.me`)
+      .then(res => res.json())
+      .then(data => setBooking(data));
+  }, []);
   const search = searchVal => {
-    console.info("TO DO!", searchVal);
+    // console.info("TO DO!", searchVal);
+    searchVal = searchVal.toLowerCase();
+    const filterd = booking.filter(
+      item =>
+        item.firstName.toLowerCase() === searchVal ||
+        item.surname.toLowerCase() === searchVal
+    );
+    setBooking(filterd);
+  };
```

Comment:
good, your filter function works well & good consideration of case too :) Something to consider for an improvement, if I search one name, 'john', followed by another 'judi', I don't get any results showing up (as the filtered list for john is in state when I search for judi) - can you have at think of a way this could be solved?